programs/x86_64/rbw

```diff
@@ -45,6 +49,7 @@ if [ "$version" != "$version0" ]; then
 	cd ..
 	if [ -d ./tmp/* 2>/dev/null ]; then mv --backup=t ./tmp/*/* ./; else mv --backup=t ./tmp/* ./"$APP" 2>/dev/null || mv --backup=t ./tmp/* ./; fi
 	chmod a+x ./"$APP" || exit 1
+	chmod a+x ./"$APPAGENT" || exit 1
```

Comment:
this isn't needed and will likely cause the scirpt to fail since iirc AM won't ask you for elevated rights when updating.